x/subkeys/internal/types/basic_fee.go

```diff
@@ -0,0 +1,47 @@
+package types
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/subkeys/exported"
+)
+
+// BasicFeeAllowance implements FeeAllowance with a one-time grant of tokens
+// that optionally expires. The delegatee can use up to SpendLimit to cover fees.
+type BasicFeeAllowance struct {
+	// SpendLimit is the maximum amount of tokens to be spent
+	SpendLimit sdk.Coins
+
+	// Expiration specifies an optional time or height when this allowance expires.
+	// If Expiration.IsZero() then it never expires
+	Expiration ExpiresAt
```

Comment:
If I use `int64` it is unclear which it is. The difference between expiring at time 1571645894 (now - from `date +'%s'`) or height 1571645894 is huge. 

If you prefer, I can use `int64` seconds instead of `time.Time` inside of `ExpiresAt`. However, I am always comparing with `ctx.BlockTime()`, which is `time.Time`, so it made sense to stay in sync with this one.

Reply:
Let's go with int64 with time then.